src/csharp/Microsoft.Spark/SparkContext.cs

```diff
@@ -299,6 +300,19 @@ public void SetCheckpointDir(string directory)
             _jvmObject.Invoke("setCheckpointDir", directory);
         }
 
+        /// <summary>
+        /// Broadcast a read-only variable to the cluster, returning a Microsoft.Spark.Broadcast
+        /// object for reading it in distributed functions. The variable will be sent to each 
+        /// executor only once.
+        /// </summary>
+        /// <typeparam name="T">Type of the variable being broadcast</typeparam>
+        /// <param name="value">Value/variable to be broadcast</param>
```

Comment:
What is `Value/variable`?

Reply:
Changed the wording to 'Value of the broadcast variable'. Please let me know if that looks fine. Thanks!